src/tools/jetbrains-plugin.md

```diff
@@ -89,8 +89,8 @@ Here's one way to configure Dart support:
   </p>
 
   <p>
-    For example, the SDK path might be
-    <code><em>&lt;dart installation directory></em>/dart/dart-sdk</code>.
+    For example, the SDK path might be (if you installed via homebrew)
```

Comment:
I don't think the majority of users are using macOS. (Analytics should be able to tell us.) Maybe we should show two or three common paths?

Reply:
Actually, the majority (~45%) are on Windows. Then macOS, and then a small share on Linux.

Reply:
Do we know the path that the majority of Windows users are likely to have?

Reply:
Sorry, no, I think that varies a lot depending on how they install it.

Reply:
@legalcodes maybe you could ask on Gitter?

Reply:
Great suggestion re: showing multiple common paths. Will find out the most common path for Windows setup. Thanks!